config/main.py

```diff
@@ -5324,6 +5384,83 @@ def naming_mode_alias():
     """Set CLI interface naming mode to ALIAS (Vendor port alias)"""
     set_interface_naming_mode('alias')
 
+@interface.group('storm-control')
+@click.pass_context
+def storm_control(ctx):
+    """ Configure storm-control"""
+    pass
+
+@storm_control.group('broadcast')
+def broadcast():
+    """ Configure broadcast storm-control"""
+    pass
+
+@broadcast.command('add')
+@click.argument('port_name', metavar='<port_name>', required=True)
```

Comment:
Please also include support for Multi-ASIC platform. ASIC name space support is required

Reply:
Please point me to an example on how to include ASIC name space.

Reply:
look into the following PR which supports multi-asic:
https://github.com/Azure/sonic-utilities/pull/1574

Reply:
@gechiang Please review